main.tf

```
@@ -1148,6 +1148,7 @@ resource "aws_s3_bucket_public_access_block" "this" {
   block_public_policy     = var.block_public_policy
   ignore_public_acls      = var.ignore_public_acls
   restrict_public_buckets = var.restrict_public_buckets
+  skip_destroy            = true
```

Replace the hardcoded value with a variable:

```suggestion
  skip_destroy            = var.skip_destroy
```
Add variable declaration in variables.tf

---

Made an update in accordance with your suggestions.

Thanks for the review ;)

---

May I get your approval in here? @paikend